config/routes.rb

```diff
@@ -82,6 +82,7 @@
   get "/docs/test-analytics/js-collectors",       to: redirect("/docs/test-analytics/javascript-collectors")
   get "/docs/tutorials/elastic-ci-stack-aws",     to: redirect("/docs/agent/v3/elastic-ci-aws")
   get "/docs/agent/clusters",                     to: redirect("/docs/clusters/overview")
+  get "/docs/apis/graphql/graphql-cookbook",      to: redirect("/docs/apis/graphql/cookbooks/pipelines")
```

Comment:
I think we should keep an overview page at `/docs/apis/graphql/graphql-cookbook`. This means you can still talk about "the cookbook" and have one link to share. It also avoids the need for a redirect.

Reply:
Makes sense. So that page would essentially have links out to all of these sub-pages?

Reply:
Yeah, it would mention what a cookbook is and link out to the child pages. Similar to: https://buildkite.com/docs/agent/v3/cli-reference

I can help with the content for that page